plugin/ochttp/client.go

```diff
@@ -69,7 +77,7 @@ func (t *Transport) RoundTrip(req *http.Request) (*http.Response, error) {
 	// TODO: remove excessive nesting of http.RoundTrippers here.
 	format := t.Propagation
 	if format == nil {
-		format = defaultFormat
+		format = compatibilityFormat
```

Comment:
Why this change?

Reply:
I just renamed the variable `defaultFormat` to `compatibilityFormat` to make it clear that we intend the new default to be TraceContext.